src/styles/generic/_links.scss

```diff
@@ -16,22 +16,28 @@ a,
 :visited {
   color: $LINK_COLOR;
   text-decoration: none;
-}
 
-a:hover,
-:link:hover,
-:visited:hover {
-  text-decoration: underline;
+  &:focus,
+  &:active,
+  &:hover {
```

Comment:
We may want to make it so hovering and pressing a link removes its underline so there's some signal to the user that it's working.